velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -807,6 +807,34 @@ struct MakeYMIntervalFunction {
   }
 };
 
+template <typename T>
+struct MakeDTIntervalFunction {
```

Comment:
Consider adding a separate file for this implementation as the DateTimeFunctions.h is large.